index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>NeoDev UI</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+    <div id="app" class="font">
+      <div class="header">
+        <div class="header_logo">
+          <a href="#"><h1>NeoDev UI</h1></a>
+        </div>
+        <div class="header_links">
```

Comment:
This is just a suggestion but you can name the div class to be header_links_container so it can be differentiated from header_links

Reply:
Okay.